python/ray/tests/test_unavailable_actors.py

```diff
@@ -70,7 +70,6 @@ def sigkill_actor(actor):
     ["actor", "task", "driver"],
 )
 @pytest.mark.skipif(sys.platform == "win32", reason="does not work on windows")
-@pytest.mark.parametrize("ray_start_regular", [{"log_to_driver": False}], indirect=True)
```

Comment:
why is it removed?

Reply:
I don't think this is useful. it disables logs to the driver, but those logs are useful for debugging